projectHelpers.js

```diff
@@ -38,13 +38,27 @@ const isVue = ({ projectDir, packageJson } = {}) => {
 
 const getPackageJson = projectDir => {
     const packageJsonPath = getPackageJsonPath(projectDir);
-    return JSON.parse(readFileSync(packageJsonPath, "utf8"));
+    return JSON.parse(fs.readFileSync(packageJsonPath, "utf8"));
 };
 
 const writePackageJson = (content, projectDir) => {
     const packageJsonPath = getPackageJsonPath(projectDir);
-    writeFileSync(packageJsonPath, JSON.stringify(content, null, 2))
+    const currentJsonContent = fs.readFileSync(packageJsonPath);
+    const indentation = getIndentationCharacter(currentJsonContent);
+    const stringifiedContent = JSON.stringify(content, null, indentation);
+    const currentPackageJsonContent = JSON.parse(currentJsonContent);
+
+    if (JSON.stringify(currentPackageJsonContent, null, indentation) !== stringifiedContent) {
+        fs.writeFileSync(packageJsonPath, stringifiedContent)
+    }
 }
+
+const getIndentationCharacter = (jsonContent) => {
+    const defaultIndentation = "  ";
+    const matches = jsonContent.match(/{\r*\n*(\W*)"/m);
+    return matches && matches[1];
```

Comment:
`return (matches && matches[1]) || defaultIndentation `

Reply:
In fact we do not need default indentation, I've forgotten to remove the variable. I've already added a unit test that verifies we work correctly when the matches[1] group is undefined.